production/helm/loki/CHANGELOG.md

```diff
@@ -25,6 +29,9 @@ Entries should include a reference to the pull request that introduced the chang
 ## 5.8.10
 
 - [ENHANCEMENT] Canary labelname can now be configured via monitoring.lokiCanary.labelname
+## 5.8.10
```

Comment:
@MattSurabian You left your previous 5.8.10 header in. This is now a duplicate and needs to be removed.

Reply:
Sorry about that!